### ISSUE TITLE
The RUV-III with PRPS Vignette for non-TCGA RNA-seq data

### ISSUE THREAD

**Issue by zy8281263** [RMolania/TCGA_PanCancer_UnwantedVariation#4]
Hi RMolania! Thanks for the very useful approach. I want to use RUV-III with PRPS for non-TCGA RNA-seq data, but I don't know how to process the non-TCGA RNA-seq data to use RUV-III with PRPS. Would you provide a Vignette for non-TCGA RNA-seq data?

Best,
Yuan

**Comment by RMolania** [comment]
Hi Yuan, 
Thank you for showing interest in our approach. We are currently in the final stages of preparing an R package for RUV-III-PRPS, and we encourage you to stay updated for its release. This package will offer seamless usability with any expression data, making it highly versatile and accessible.
All the best,
Ramyar

**Comment by zy8281263** [comment]
您好，我已收到您的邮件。我将尽快给您回复。

**Comment by kai0511** [comment]
Hi, Ramyar, when will the package for RUV-III-PRPS be released? I am looking at the vignettes of the repository to know how to use it, but it seems less intuitive to me. 

An another question, if I am not intended to remove variation of tumor purity, can I still use the tool for TCGA?

Thanks!

Kai

**Comment by zy8281263** [comment]
您好，我已收到您的邮件。我将尽快给您回复。

**Comment by kai0511** [comment]
> Hi Yuan,
> Thank you for showing interest in our approach. We are currently in the final stages of preparing an R package for RUV-III-PRPS, and we encourage you to stay updated for its release. This package will offer seamless usability with any expression data, making it highly versatile and accessible.
> All the best,
> Ramyar

Hi, Ramyar, when will the package for RUV-III-PRPS be released? I am looking at the vignettes of the repository to know how to use it, but it seems less intuitive to me.

An another question, if I am not intended to remove variation of tumor purity, can I still use the tool for TCGA?

Thanks!

Kai

**Comment by EzequielD-86** [comment]
Agreed, this packages seems like it is exactly what I need for patient dataset, a vignette for non TCGA data would be greatly appreciated!

**Comment by zy8281263** [comment]
您好，我已收到您的邮件。我将尽快给您回复。
